pkg/resource/deploy/step.go

```diff
@@ -688,7 +687,7 @@ func (op StepOp) Prefix() string {
 func (op StepOp) RawPrefix() string {
 	switch op {
 	case OpSame:
-		return "  "
+		return "= "
```

Comment:
We made a particular decision to not have any glyph for 'sameness' so as to not clutter the display too much.  @joeduffy